eqc/mstore_functions_eqc.erl

```diff
@@ -24,7 +24,7 @@ offset() ->
 %%%-------------------------------------------------------------------
 %%% Properties
 %%%-------------------------------------------------------------------
-prop_cout_parts() ->
+prop_count_parts() ->
     ?FORALL({Size, Time, Count}, {size(), offset(), length()},
             lists:sum([C || {_, C} <-mstore:make_splits(Time, Count, Size)]) == Count).
```

Comment:
Line 29 is too long:             lists:sum([C || {_, C} <-mstore:make_splits(Time, Count, Size)]) == Count)..